_posts/2025-05-16-perforce-eula.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: The Perforce EULA and impact on Vox Pupuli
+date: 2025-05-16
+github_username: bastelfreak
+---
+
+In November 2024, [Perforce announced](https://www.puppet.com/blog/open-source-puppet-updates-2025) that they will stop providing most tools & projects as open source.
```

Comment:
We might consider mentioning that while the repos remain available and unarchived, they've made it clear they are not going to maintain them at all and we've seen zero evidence of backporting any fixes from Puppet Core whatsoever.

Reply:
I'm a bit careful here. They also said that the apt/yum repos are read only but then they still published new puppet-release packages with an updated gpg key. I've zero confidence anymore in statements from perforce.